go/test/endtoend/vreplication/migrate_test.go

```diff
@@ -165,3 +167,155 @@ func TestMigrate(t *testing.T) {
 		require.Errorf(t, err, "there is no vitess cluster named ext1")
 	})
 }
+
+// TestVtctldMigrate runs an e2e test for importing from an external cluster using the vtctld Mount and Migrate commands.
+// We have an anti-pattern in Vitess: vt executables look for an environment variable VTDATAROOT for certain cluster parameters
+// like the log directory when they are created. Until this test we just needed a single cluster for e2e tests.
+// However now we need to create an external Vitess cluster. For this we need a different VTDATAROOT and
+// hence the VTDATAROOT env variable gets overwritten.
+// Each time we need to create vt processes in the "other" cluster we need to set the appropriate VTDATAROOT
+func TestVtctldMigrate(t *testing.T) {
```

Comment:
Is this any different, aside from the `vc.VtctldClient` usage? If not, then IMO it would be better to make the client to use a parameter to a shared Test function.